Header header
Sequence seq
uint8[] pins_to_signal
---
Header header
Sequence seq
uint8[] pins_to_signal
